projects/angular/src/data/datagrid/datagrid-items-trackby.ts

```diff
@@ -16,6 +17,17 @@ export class ClrDatagridItemsTrackBy<T = any> {
 
   @Input('ngForTrackBy')
   set trackBy(value: TrackByFunction<T>) {
+    /**
+     * This is a workaround to prevent the items `trackBy` function from
+     * being replaced when the "manage columns" button is clicked. This is
+     * not a complete solution. If there is another `ngForTrackBy` function
+     * within the datagrid in application code, it could sill replace the
+     * items `trackBy` function whether it is the a row iterator or not.
```

Comment:
`whether it is the a row iterator or not.` -> `whether it is the row iterator or not.`

Reply:
Oops. I will fix it. Thanks!